tests/lib/prepare.sh

```diff
@@ -233,6 +233,51 @@ linux /vmlinuz root=$ROOT ro init=$IMAGE_HOME/reflash.sh console=ttyS0
 initrd /initrd.img
 }
 EOF
+        # prepare block device on swap for cold-plug system-user autoimport testing
+        # deactivate swap
+        swapoff -a
+        # create new primary partition occupying all the /dev/sdb disk
+        echo -e "o\nn\np\n1\n\n\nw" | fdisk /dev/sdb
```

Comment:
I'm amazed that after several years since I used it first this is still the best way to create partitions in scripts. May forth backwards school live on :-)  (nothing against your work Federico, just an observation)